sdk/core/core-http/rollup.base.config.js

```diff
@@ -87,7 +87,7 @@ export function browserConfig(test = false, production = false) {
         namedExports: {
           chai: ["assert", "AssertionError", "should", "expect"],
           events: ["EventEmitter"],
-          "@opentelemetry/types": ["CanonicalCode", "SpanKind", "TraceFlags"]
+          "@opentelemetry/api": ["CanonicalCode", "SpanKind", "TraceFlags"]
```

Comment:
How is it that there is no corresponding update to rollup config for app config?

Reply:
Strangely, rollup doesn't seem to need the named exports config for OpenTelemetry. I'm not sure why. /cc @bterlson

Reply:
appconfig doesn't have rollup for browser. Not sure why though. @richardpark-msft